service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManagerOptions.java

```diff
@@ -37,4 +38,16 @@
     @Getter
     @Builder.Default
     private final int httpConnectTimeout = DEFAULT_HTTP_CONNECT_TIMEOUT_MS;
+
+    /**
+     * The authentication scopes to use when requesting authentication tokens from Azure Active Directory for
+     * authenticating with IoT Hub. This value is only used by the client if the client is configured to use role-based
+     * access credentials rather than symmetric key based credentials. This value defaults to the authentication scopes
+     * that are used for all public cloud deployments and all private cloud deployments other than those in the
+     * Fairfax cloud. For Fairfax cloud users, this value must be set to
+     * {@link IotHubAuthenticationScopes#FAIRFAX_AUTHENTICATION_SCOPES}.
```

Comment:
I thought Fairfax was an internal code name for US Government Cloud. Should we use this term externally?

Reply:
Good question. I'll run this by some service PMs to get their take on it